### DISCUSSION TITLE
"Could not fetch total commits"

### DISCUSSION THREAD

**Discussion by gabrielrbarbosa** [anuraghazra/github-readme-stats#3302] [Q&A] [answered]
### Describe the bug

Card is displaying the following message:
"Something went wrong! file an issue at https://tiny.one/readme-stats. Could not fetch total commits"

### Expected behavior

Show readme stats card

### Additional context

https://camo.githubusercontent.com/7b5952d7a6518687537fffe1a9cfd4bc97f3ae1d7fc002d01e4e3adc6ea7cf8a/68747470733a2f2f6769746875622d726561646d652d73746174732e76657263656c2e6170702f6170693f757365726e616d653d6761627269656c72626172626f736126696e636c7564655f616c6c5f636f6d6d6974733d74727565267468656d653d616c676f6c69612673686f775f69636f6e733d74727565

**Comment by Prasad-mutnale** [comment]
hello @gabrielrbarbosa i would like to contribute to this issue please i can resolve this issue efficient way please assign me this issue

**Reply by qwerty541** [reply]
@Prasad-mutnale Unfortunately, you cannot influence the periodic occurrence of this problem in any way since it is related to the upstream GitHub API. Unless you are a GitHub employee.

**Comment by qwerty541** [comment, marked as answer]
Hey, @gabrielrbarbosa! You can see this when upstream GitHub API respond with error due to some reason during fetching stats card data. It should be resolve automatically after some time. Recently we merged https://github.com/anuraghazra/github-readme-stats/pull/3238 which improved confusing behavior of showing 0 commits, currently we show error card instead.

**Comment by LucasDGS** [comment]
I am having trouble fetching all the commits, it seems like it is fetching the commits from the past few days, instead of the entire year, has anyone else experienced this?

`api?username=LucasDGS&show_icons=true&theme=default&hide=stars&include_all_commits=true)]`

(https://github.com/anuraghazra/github-readme-stats)
![image](https://github.com/anuraghazra/github-readme-stats/assets/88339876/abe0157b-aab8-49ac-8f21-4b789f61e0a6)

**Reply by qwerty541** [reply]
Hey, @LucasDGS! By default, the card only shows statistics from public repositories. To show private statistics on the card, you should [deploy your own instance](https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own) using your own GitHub API token. Our documentation contains that information.

![image](https://github.com/anuraghazra/github-readme-stats/assets/53787217/d2745ae3-8a5b-47f5-b776-9d0594c2e60f)
